### PR TITLE
Allow user to override what scheme to run

### PR DESCRIPTION
**Pull request by markx**

### CODE REVIEW

**Comment by Olical** [comment]
Oh this is cool! I only used this repo for SICP exercises, hence the mit-scheme. I wonder if there is a nice way to make the mapping filetypes configurable too 🤔

**Comment by markx** [comment]
Yeah I also plan to use this for SICP, but I'll do it in Racket.

Racket is a popular choice for SICP. I don't think many scheme dialects have their own filetypes.

File: autoload/scheme.vim [hunk 1]
```diff
@@ -1,6 +1,11 @@
 function! scheme#connect()
   new
-  let s:repl_term_id = termopen('mit-scheme')
+
+  if !exists('g:scheme_executive')
+    let g:scheme_executive = "mit-scheme"
```
Comment (Olical) [inline autoload/scheme.vim]: Ah, by `executive`, do you mean `executable`? I think that's the common name for this sort of thing?

**Comment by Olical** [comment]
Maybe we should rename that variable to `executable` then I'm more than happy to merge, this is a good change!

**Comment by markx** [comment]
Updated. Please take another look.

**Review by Olical — APPROVED** [review]
Awesome, thank you very much!

**Pull request merged**